data/data_test.go

```diff
@@ -0,0 +1,132 @@
+package data
```

Comment:
Using `package data_test` prevents collisions with test fixtures. For instance if `NewClient` was defined here as well as in `data.go`.

Reply:
That makes sense -- I have done this when using Ginkgo, but I wasn't sure if it was a general practice.

Reply:
I prefer it. Using `package data_test` makes it to where you have to test only what is exported by the package. I only use something like `package data` when I explicitly want to test a function/type that is not exported by the package. In that situation I often consider just creating an `internal` package. See: https://docs.google.com/document/d/1e8kOo3r51b2BWtTs_1uADIA5djfXhPT36s6eHVRIvaU/edit